app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt

```diff
@@ -903,6 +927,10 @@ class WebViewActivity : BaseActivity(), io.homeassistant.companion.android.webvi
             checkAndWarnForDisabledLocation()
         }
         changeLog.showChangeLog(this, false)
+
+        if (::loadedUrl.isInitialized) {
+            reloadPage()
+        }
```

Comment:
Please revert. This will cause a reload with state loss anytime the app is resumed, eg. when opening it from the background.

Reply:
`reloadPage` reloads when there is an ERROR + run `waitForConnection`. Only `waitForConnection` does not work on error.

If the page is normal, then there is no reload. This is needed when returning from Settings, etc.